nix/formatters.nix

```diff
@@ -0,0 +1,20 @@
+# This file is part of the IOGX template and is documented at the link below:
+# https://www.github.com/input-output-hk/iogx#38-nixformattersnix
+
+{
+  stylish-haskell.enable = true;
+  cabal-fmt.enable = true;
+  shellcheck.enable = true;
+  editorconfig-checker.enable = true;
+  nixpkgs-fmt.enable = true;
+  png-optimization.enable = true;
+
+  # TODO
+  # open a new PR
+  # set this to true
+  # run 'pre-commit run hlint --all-files'
+  # fix everything up
+  # push & merge
+  # remove this comment
+  hlint.enable = false;
```

Comment:
I deleted `.github/workflows/hlint.yml`, and `.github/.hlint.yml`, which was just:
```
- ignore: { }
```
Which is a no-op, always, right?

By setting `hlint.enable = true;` in `./nix/formatters.nix` instead, `hlint` will be run (using the top-level `.hlint.yml` file) with each `git commit`, and also manually with `pre-commit run hlint` and also automatically run in CI along with all other formatters.

But because pre-commit only works on committed files, in order to lint existing .hs sources, you will have to run `pre-commit run hlint --all-files` and fix everything up (which you'd wanna do in a separate PR).
Thoughts?